app/actions/v3/service_binding_create.rb

```diff
@@ -72,6 +72,7 @@ def save_failed_state(binding, e)
             description: e.message,
           }
         )
+        binding
```

Comment:
`save_failed_state` is always called before an exception is raised, so this returned binding is unused.